test/suites/optional/multus.robot

```diff
@@ -0,0 +1,139 @@
+*** Settings ***
+Documentation       Tests for Multus and Bridge plugin on MicroShift
+
+Resource            ../../resources/common.resource
+Resource            ../../resources/microshift-process.resource
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+
+*** Variables ***
+${BRIDGE_INTERFACE}     br-test
+${NAD_YAML}             ./assets/multus/bridge-nad.yaml
+${POD_YAML}             ./assets/multus/bridge-pod.yaml
+${POD_NAME}             test-bridge
+${POD_NS}               default
+
+
+*** Test Cases ***
+Pre-Existing Bridge Interface
+    [Documentation]    Test verifies if Bridge CNI plugin will
+    ...    work correctly with pre-existing interface.
+    [Setup]    Run Keywords
+    ...    Recreate Fresh Interface    ${BRIDGE_INTERFACE}
+    ...    AND
+    ...    Setup Bridge Test
+
+    Curl Pod Over Bridge Interface    ${POD_NAME}    ${POD_NS}    ${BRIDGE_INTERFACE}
+
+    [Teardown]    Remove NAD And Pod
+
+No Pre-Existing Bridge Interface
+    [Documentation]    Test verifies if Bridge CNI plugin will
+    ...    work correctly if there is no pre-existing bridge
+    ...    interface - it needs to be created.
+    [Setup]    Run Keywords
+    ...    Make Sure Interface Does Not Exist    ${BRIDGE_INTERFACE}
```

Comment:
Yes, though it'll need two (or single templated) NetworkAttachmentDefinition and Pod manifests.
We'll have similar problem with ipvlan / macvlan tests, because:
> A single master interface can not be enslaved by both macvlan and ipvlan. ([src](https://www.cni.dev/plugins/current/main/ipvlan/#notes))

Perhaps we want to add more interface to the VM to avoid that?

Reply:
Would we need more than 2? The bridge test doesn't need a new one, right?

Reply:
No, bridge plugin doesn't have this exclusivity rule. Other solution is to (probably) reboot the host after the test, so the other *vlan can enslave main interface

Reply:
Which approach seems like it will be most reliable?

Reply:
I don't have data on that, but adding two nics will be faster (if we keep random order of the tests, then we'll have two reboots).

Reply:
Adding the extra NICs sounds like a good place to start, then.